app/main/controller/similarity_async_controller.py

```diff
@@ -6,8 +6,8 @@
 from app.main.lib import similarity
 from app.main.lib.webhook import Webhook
 
-api = Namespace('similarity_sync', description='synchronous similarity operations')
-similarity_sync_request = api.model('similarity_sync_request', {
+api = Namespace('similarity_async', description='synchronous similarity operations')
```

Comment:
should we change the description too? "asynchronous similarity operations"

Reply:
Good call! Yes, let's fix the description accordingly please.